### PR TITLE
Drop codecov USAGE by this package - no longer free!

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by natebosch** [comment]
I think we should drop support in the package if we are not testing the support ourselves. How do we expect to debug once it breaks?

File: mono_repo.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ merge_stages:
 
 coverage_service:
 - coveralls
-- codecov
+#- codecov - dropping because it's no longer free!
```
Comment (jakemac53) [inline mono_repo.yaml:7]: Should we just delete this?

**Review by jakemac53 — APPROVED** [review]

File: mono_repo.yaml (hunk 1 above)
Comment (kevmoo) [reply mono_repo.yaml:7]: I guess?

**Comment by kevmoo** [comment]
nevermind. it works now!

**Pull request closed without merging**